src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -115,12 +114,17 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
                 ? await FindElementAccessReferencesAsync(symbol, document, semanticModel, options, cancellationToken).ConfigureAwait(false)
                 : ImmutableArray<FinderLocation>.Empty;
 
+            var conditionalAccessReferences = symbol.IsIndexer
+                ? await FindConditionalAccessReferencesAsync(symbol, document, semanticModel, options, cancellationToken).ConfigureAwait(false)
+                : ImmutableArray<FinderLocation>.Empty;
+
             var indexerCrefReferences = symbol.IsIndexer
                 ? await FindIndexerCrefReferencesAsync(symbol, document, semanticModel, options, cancellationToken).ConfigureAwait(false)
                 : ImmutableArray<FinderLocation>.Empty;
```

Comment:
can we move all of the `.IsIndexer` work into a single `FindIndexerReferences` call which breaks out into the different types of references we might have?

Reply:
This was solved together with changes to remove code duplication.